### PR TITLE
Update the score weights of all contribution types

### PR DESCRIPTION
**Pull request by aanchal-adk**
### Description
- Move the score weight declarations to .env
- Update score weights of all contributions types
- Remove 'Repo created' column from the score table 
- Rename 'Issue comments' to 'PR/Issue comments' as the score is calculated on the basis of both comment types

### Screenshots
Screenshot from 2020-10-14 11-42-07

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -5,3 +5,28 @@ USER_LIST=user-name1, user-name2
...
+T3S2=0.3
+
+# score weights for various sub-type of 'PR/Issue comments' type
+T4S1=0.1
+T4S2=0.1
+T4S3=0.1
+T4S4=0.1
+
+# score weight for 'Repo created' type
+T5S1=0
```
Comment (kabirbaidhya) [inline .env.example:32]:
```suggestion
T5S1=0

```

File: leaderboard/utils/final_score_table.py [hunk 2]
```diff
@@ -1,8 +1,23 @@
...
+score_wt_T2S3 = float(os.environ.get("T2S3", 1))
+score_wt_T2S4 = float(os.environ.get("T2S4", 1))
+score_wt_T3S1 = float(os.environ.get("T3S1", 1))
+score_wt_T3S2 = float(os.environ.get("T3S2", 1))
+score_wt_T4S1 = float(os.environ.get("T4S1", 1))
+score_wt_T4S2 = float(os.environ.get("T4S2", 1))
+score_wt_T4S3 = float(os.environ.get("T4S3", 1))
+score_wt_T4S4 = float(os.environ.get("T4S4", 1))
+score_wt_T5S1 = float(os.environ.get("T5S1", 1))
+
```
Comment (kabirbaidhya) [inline leaderboard/utils/final_score_table.py:20]: These actually doesn't have to come through an env file. Can you get it from a `config.py`? These are the weights needed by algorithm itself and not a part of env. We can have an ability of environment override later if needed, but I don't think we need it.

File: .env.example (hunk 1 above)
Comment (kabirbaidhya) [reply .env.example:32]: On the second thought, we don't need these in the envrionment. These are a part of internal configurations.

Comment (aanchal-adk) [reply .env.example]: @kabirbaidhya I've reverted it back to how it was and just changes the score weights.

File: leaderboard/utils/final_score_table.py (hunk 2 above)
Comment (aanchal-adk) [reply leaderboard/utils/final_score_table.py]: done

Comment (aanchal-adk) [reply leaderboard/utils/final_score_table.py]: done

**Review by mesaugat — APPROVED** [review]

**Review by bhattaraib58 — APPROVED** [review]

**Review by kabirbaidhya — APPROVED** [review]

**Pull request merged**